CMakeLists.txt

```diff
@@ -101,16 +101,25 @@ endif()
 # Installation logic...
 #-----------------------------------------------------------------------------------------------
 
+if (NOT APPLE AND NOT WIN32)
```

Comment:
If this is for linux only then specify `if(LINUX)` instead of `NOT OTHER STUFF`?

Reply:
There is no such definition as `LINUX`. The definition `UNIX` does exist. However, it is set for both Linux and for `APPLE`, too. Either `if (NOT APPLE AND NOT WIN32)` or `if(UNIX AND NOT APPLE)`.

Reply:
How about we put at the top level cmake file
```
if (UNIX AND NOT APPLE)
    set(LINUX TRUE)
endif()
```
Then we can do the `if (LINUX)`